resources/assets/sass/app.scss

```diff
@@ -1,63 +1,112 @@
 $font-size-base: 20px;
+$font-family-base: maison-neue, Roboto, sans-seif;
+$body-bg: #F5F6F7;
 
 @import "node_modules/bootstrap-sass/assets/stylesheets/bootstrap";
 
 body {
-    padding-bottom: 4rem;
+    color: #50636C;
+    line-height: 30px;
+    font-weight: 300;
+}
+
+p {
+  margin-bottom: 60px;
 }
 
 img {
-    max-width: 100%;
+  max-width: 100%;
 }
 
-.page-title {
-    clear: both;
+header {
+    background: #fff;
 }
 
-.giscus-logo {
-    clear: both;
-    margin: 4rem 0 3rem;
-    max-width: 40rem;
+.ribbon {
+    background: linear-gradient(-90deg, #D7516A 0%, #479CC3 100%);
+    height: 12px;
+}
 
-    @media only screen and (max-width: 600px) {
-        width: 75%;
+.navbar-head {
+    &.navbar {
+        margin-bottom: 0;
     }
-}
 
-.pitch {
-    font-size: 1.3em;
-    font-weight: bold;
-    line-height: 1.1;
+    .navbar-brand {
+        height: auto;
+        margin: 50px 0;
+        padding: 0;
+    }
 
-    @media only screen and (min-width: 600px) {
-        font-size: 1.5em;
+    .navbar-buttons {
+        text-align: right;
+    }
+    .navbar-buttons .btn {
+        margin: 50px 0;
     }
 }
 
-.pricing {
-    font-size: 1.25em;
-    margin: 1em 0 1.5em;
+.main-content {
+    padding: 4rem 0;
+}
+
+.welcome-heading {
+    margin-bottom: 60px;
+    font-size: 36px;
+    color: #1C313B;
+    line-height: 44px;
+    font-weight: 400;
```

Comment:
Does the font we're using have the difference between these various font weights?